### PR TITLE
[hotfix] Add missing configurations to flink session cluster deployment

### PR DESCRIPTION
**Pull request by sjwiesman**

### CODE REVIEW

File: docs/deployment-and-operations/packaging.md [hunk 1]
```diff
@@ -60,12 +60,19 @@ If you prefer to package your job to submit to an existing Flink cluster, simply
...
 <div class="alert alert-info">
   <strong>Attention:</strong> The distribution must be bundled in your application fat JAR so that it is on Flink's <a href="https://ci.apache.org/projects/flink/flink-docs-stable/monitoring/debugging_classloading.html#inverted-class-loading-and-classloader-resolution-order">user code class loader</a>
 </div>
 
 {% highlight bash %}
-./bin/flink run ./statefun-example.jar
+./bin/flink run -c org.apache.flink.statefun.flink.core.StatefulFunctionsJob ./statefun-example.jar
 {% endhighlight %}
+
+You will also need to add the following two configurations to your clusters `flink-conf.yaml`.
```
Comment (tzulitai) [inline docs/deployment-and-operations/packaging.md:72]: Could we rephrase this as:
these configurations are strictly required for running Statefun applications, or something along those lines?

As such, the max concurrent checkpoints should also be mentioned here.

**Comment by sjwiesman** [comment]
Merging
